error-handling-proto/build.gradle.kts

```diff
@@ -2,29 +2,20 @@ description = "Transform dead letters in Kafka Streams applications to protobuf.
 
 plugins {
     id("java-library")
-    id("com.google.protobuf") version "0.9.4"
+    alias(libs.plugins.protobuf)
 }
 
-val protobufVersion: String by project
+val protobufVersion = libs.protobuf.get().version
```

Comment:
Is this still required?

Reply:
Yes, we use it below to configure protoc